src/perf_tool_backend.ml

```diff
@@ -376,8 +392,34 @@ module Perf_line = struct
           "Regex of trace error did not match expected fields" (results : string array)]
   ;;
 
-  let ok_perf_line_to_event line ~(perf_map : Perf_map.t option) : Event.Ok.t =
-    match Re.Group.all (Re.exec ok_perf_line_re line) with
+  let ok_perf_power_line_to_event matches : Event.Ok.t option =
+    match matches with
+    | [| _; pid; tid; time_hi; time_lo; _; kind; _; freq; _ |] ->
+      let pid = Int.of_string pid in
+      let tid = Int.of_string tid in
+      let time = parse_time ~time_hi ~time_lo in
+      (match kind with
+      | "cbr" ->
+        (* cbr (core-to-bus ratio) are events which show frequency changes. *)
+        Some
+          (Power
+             { thread =
+                 { pid = (if pid = 0 then None else Some (Pid.of_int pid))
+                 ; tid = (if tid = 0 then None else Some (Pid.of_int tid))
+                 }
+             ; time
+             ; freq = Int.of_string freq
+             })
+      | _ -> None)
```

Comment:
Explicitly list "psb offs" here, and raise if we get anything else.

Reply:
Added this.